src/utils/getArticleList.js

```diff
@@ -0,0 +1,14 @@
+import thingsApi from "./axiosSettings";
+
```

Comment:
In this case we can probably create a file that handles all the different data fetches, rather than having a file for each.